Advance-1/api.php

```diff
@@ -0,0 +1,114 @@
+<?php 
+require '../vendor/autoload.php';
+
+use GuzzleHttp\Client;
+
+/**
+ * Class for extracting data from the API.
+ */
+class API
+{
```

Comment:
Please use proper naming convention.  Class name must start with a verb. Please correct the position of the bracket  .Also leave a line gap after the starting of a class.

Reply:
Done.